### ISSUE TITLE
Issue on `html` prop

### ISSUE THREAD

**Issue by ai** [pheuter/svelte-check-rs#5]
For some reason, parser do not allow me to use my components with `html` props.

```svelte
<script lang="ts">
    import Other from "./other.svelte";
</script>

<Other html="HTML to escape" />
```

```sh
$ pnpm svelte-check
Loading svelte-check in workspace: /workspaces/svelte-check-rs-test
Getting Svelte diagnostics...

svelte-check found 0 errors and 0 warnings

$ ~/.cargo/bin/svelte-check-rs
web/src/test.svelte:5:8
Error: unexpected token: expected '>', found 'html' (parse-error)

web/src/test.svelte:6:1
Error: unclosed tag: <Other> (parse-error)

====================================
svelte-check found 2 errors and 0 warnings in 2 files
```

Reproduction: https://github.com/ai/svelte-check-rs-test/tree/html-prop
